libraries/lib-datahandler/src/main/thrift/users.thrift

```diff
@@ -54,6 +54,7 @@ struct User {
     9: optional string lastname, // lastname or surname of the person
     10: required string department,
     11: optional bool wantsMailNotification,
+    12: optional string commentMadeDuringModerationRequest,
```

Comment:
Alright, as discussed, the alternative of adding the comment parameter to all methods that have something to with saving projects/components/releases/etc is even uglier. We'll let the field stay. However, please make sure that this fields never lands in the DB. The place to do this is `ThriftValidate.prepareUser()`. (Turns out we have a pre-existing error that this method is never used.)